chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/Validator.java

```diff
@@ -29,11 +29,12 @@ public synchronized void validate(final CertifiedProductSearchDetails listing) {
         }
     }
 
-    public void validate(final CertifiedProductSearchDetails existingListing,
-            final CertifiedProductSearchDetails updatedListing) {
-        validate(updatedListing);
-        for (ComparisonReviewer reviewer : getComparisonReviewers()) {
-            reviewer.review(existingListing, updatedListing);
+    public void validate(CertifiedProductSearchDetails existingListing, CertifiedProductSearchDetails updatedListing) {
+        if (updatedListing.isCertificateActive()) {
```

Comment:
I think this change is great for the 2015 listings. I just remembered I had hoped to change the 2014 validation to use the "AllowedListingValidator". Not recalling the name exactly but it's whatever we use for 2011 edition. There is a validator factory class which I believe assigns the correct validator to a listing. I was hoping to basically remove all references to 2014 edition listing validators in this PR and maybe, hopefully, delete those files.